dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -545,6 +568,25 @@ public Label getLabel() {
         public String toString() {
             return "JumpMarker(" + getLabel() + ")";
         }
+
+        @Override
+        public String toStringDebug() {
+            return toString();
+        }
+    }
+
+    /**
+     * Return a printed representation of a collection of extended nodes.
+     *
+     * @param nodes a collection of extended nodes to format
+     * @return a printed representation of the given collection
+     */
+    public static String extendedNodeCollectionToString(Collection<? extends ExtendedNode> nodes) {
```

Comment:
I think it would be clearer to call this `extendedNodeCollectionToStringDebug`.  Then it's clear in `PhaseOneResult#toStringDebug` why `PhaseOneResult#toString` isn't called.